doc/articles/faq.md

```diff
@@ -1,4 +1,154 @@
+
+**For designers**, XAML provides a common way to use pixel-perfect designs and use rich UI interactions.
+
+## Why Uno ?
+Developing for Windows (phone, desktop, tablet, XBox), iOS (tablet and phone), Android (tablet and Phone) and WebAssembly at once can be a complex process, especially when it comes to the user interface. Each platform has its own ways of defining dynamic layouts, with some being more efficient, some more verbose, some more elegant, and some more performant than others.
+Yet, being able to master all these frameworks at once is a particularly difficult task, because of the amount of platform-specific knowledge required to master each platform. Most of the time it boils down to different teams developing the same application multiple times, with each requiring a full development cycle.
+With Xamarin, C# comes to all these platforms; however, it only provides transparent translations of the UI frameworks available for iOS and Android. Most non-UI code can be shared, but when it comes to the UI, almost nothing can be shared.
+To avoid having to learn the UI-layout techniques and approaches for each platform, Uno.UI mimics the Windows XAML approach of defining UI and layouts. This translates into the ability to share styles, layouts, and data-bindings while retaining the ability to mix XAML-style and native layouts. For instance, a StackPanel can easily contain a RelativeLayout on Android, or a MKMapView on iOS.
+Uno.UI provides the ability for developers to reuse known layout and coding techniques on all platforms, resulting in a gain of overall productivity when creating UI-rich applications.
+
+
+## Who makes UNO platform?
+Uno Platform was created by Team nventive over the past 4 years.
+
+## What makes UNO Platform unique?
+The Uno Platform is the only implementation of the UWP API that runs across iOS, Android and WebAssembly. 
+
+## Where does the name Uno Platform come from?
+
+## Is Uno Platform open source?
+Yes, the Uno Platform is open source, under the [Apache 2.0 license](https://github.com/nventive/Uno/blob/master/License.md).
+
+# Getting started
+## How can I try Uno platform?
+You can try uno using the [Uno Platform Playground](https://playground.platform.uno), the [Quick Start GitHub repository](https://github.com/nventive/Uno.QuickStart), or through our  [Visual Studio Extension](https://marketplace.visualstudio.com/items?itemName=nventivecorp.uno-platform-addin).
+
+## How experienced do I need to be to use Uno Platform?
+
+## What is the playground?
+The [Uno Platform Playground](https://playground.platform.uno) is a convenient way to experiment with the Uno platform, using the only a WebAssembly compatible web browser.
+
+## Where can I get support?
+Support is available through our [twitter account](https://twitter.com/unoplatform), [gitter](https://gitter.im/uno-platform/Lobby) and email [info@platform.uno](mailto:info@platform.uno).
```

Comment:
Maybe list Twitter last, since the Gitter channel should normally be the first point of call.